.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,13 @@
-### Context
+## Please fill in the sections below
 
+After you submit your issue, the technical writing team from Central Digital and Data Office (CDDO) will discuss and prioritise it at our fortnightly triage meeting. We’ll then let you know if and when we’ll move it forward.
+ 
+## What’s changed
 
-### Changes proposed in this pull request
+What are you trying to do? Is this something that changes how the Tech Docs Template behaves, or is it fixing a bug? 🐛
 
 
-### Guidance to review
+## Identifying a user need
+
+Do you have evidence whether the change will meet the needs of users? Let us know about any user research or testing you’ve done.
+
```

Comment:
Should we add a section on who should review this? e.g. FE dev, BE dev, someone in a specific team or with a specific set of skills acquired over a very long career?

Reply:
I'm happy to leave as is for this iteration and steer away from mandating anything just yet but happy to keep an eye on it to check the PRs are meeting a certain standard and tweak it in future if needed